Integrations/GitHub/GitHub.py

```diff
@@ -0,0 +1,277 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+import requests
+import json
+import collections
+
+# disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+API_KEY = ""
+INSECURE = demisto.params().get('insecure')
+BASE_URL = 'https://api.github.com/'
+
+'''HELPER FUNCTIONS'''
+
+
+def http_request(method, URL_SUFFIX, json=None):
+    if method is 'GET':
+        headers = {
+            'Content-Type': 'application/json',
+            'Authorization': 'Bearer ' + API_KEY
+        }
+    elif method is 'POST':
+        if not API_KEY:
+            headers = {
+                'Content-Type': 'application/json',
+                'Accept': 'application/json'
+            }
+        else:
+            headers = {
```

Comment:
pretty sure the headers should be the same at all times for these cases, please make that a const